### PR TITLE
Testnet 17

### PR DESCRIPTION
**Pull request by gterzian**
This PR is a batch for testnet 17. These PRs should be merged before merging this one : 

- #3202 
- #3205
- #3162
- #3203
- https://github.com/massalabs/massa/pull/3232
- #3192
- #3229 
- #3221 
- #3219 
- #3210 
- #3191 
- #3173 
- https://github.com/massalabs/massa-sc-runtime/pull/161
- https://github.com/massalabs/massa-sc-runtime/pull/164
- #3253 
- #3249 
- #3238 

After merging this PR we should merge these before release testnet 17 : 
- https://github.com/massalabs/massa-as-sdk/pull/49
- https://github.com/massalabs/massa-as-sdk/pull/39

### CODE REVIEW

**Comment by gterzian** [comment]
Target branch for work meant for testnet 17 @massalabs/core-team

**Comment by AurelienFT** [comment]
I think this should point on #3163

File: massa-node/base_config/openrpc.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "openrpc": "1.2.4",
     "info": {
         "title": "Massa OpenRPC Specification",
-        "version": "TEST.16.1",
+        "version": "TEST.17.1",
```
Comment (damip) [inline massa-node/base_config/openrpc.json]: 17.1 ?

Comment (AurelienFT) [reply massa-node/base_config/openrpc.json]: Has been fixed in the next commit.

**Review by AurelienFT — APPROVED** [review]

**Comment by AurelienFT** [comment]
bors merge

**Comment by gterzian** [comment]
@massalabs/core-team FYI tested the gas price replacement by fee on the labnet, it seems to work fine as:

1. Fee is added to block reward, split among endorser(right?)
2. Transaction is rejected if fee is too high(although it's not 100% clear, since one doesn't get feedback other than that the transaction "dissapears", as noted in https://github.com/massalabs/massa/issues/3295)

**Pull request merged**